swan-cern/scripts/before-notebook.d/10_spark.sh

```diff
@@ -26,10 +26,9 @@ then
         '{load_extensions: $ARGS.named}' > $NBCONFIG/spark.json
 
   IPYTHON_KERNEL_CONFIG=$IPYTHONDIR/profile_default/ipython_kernel_config.py
-  sed -i "1s/^/c.InteractiveShellApp.extensions.append('sparkconnector.connector')\n/" \
-      $IPYTHON_KERNEL_CONFIG
-  sed -i "2s/^/c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')\n/" \
-      $IPYTHON_KERNEL_CONFIG
+
+  echo -e "c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')\n$( [ -f ${IPYTHON_KERNEL_CONFIG} ] && cat ${IPYTHON_KERNEL_CONFIG})" > ${IPYTHON_KERNEL_CONFIG}
```

Comment:
I would do these two lines in a separate commit. This is just an improvement to be able to insert at the beginning of a file when that file is empty. This is needed because when using custom environments, the IPYTHON_KERNEL_CONFIG file has not been created beforehand. All this should appear in the commit message.